src/main/scala/faces/apps/ModelViewer.scala

```diff
@@ -21,19 +21,19 @@
  import javax.swing._
  import javax.swing.event.{ChangeEvent, ChangeListener}
  import breeze.linalg.min
+ import scalismo.color.{RGB, RGBA}
  import scalismo.faces.gui.{GUIBlock, GUIFrame, ImagePanel}
  import scalismo.faces.gui.GUIBlock._
  import scalismo.faces.parameters.RenderParameter
  import scalismo.faces.io.{MeshIO, MoMoIO, PixelImageIO, RenderParameterIO}
  import scalismo.faces.sampling.face.MoMoRenderer
- import scalismo.faces.color.RGB
  import scalismo.faces.image.PixelImage
  import scalismo.utils.Random
- import scalismo.faces.color.RGBA
  import scalismo.faces.momo.MoMo
 
  import scala.reflect.io.Path
  import scala.util.{Failure, Try}
+ import collection.{ Seq => CSeq }
```

Comment:
Could you please remove this line,
change line 45 back to using the type Seq,
and  add `.toIndexedSeq` to the variable `args `where `getModelFile` is called?

I think its totally fine if there is now an immutable Seq used in `getModelFile`.